libs/user/user.ts

```diff
@@ -186,12 +216,18 @@ class User {
   }
 
   /**
-   * Allow user to transfer tokens in Polygon Nightfall
+   * Allow user to transfer tokens in Nightfall to other Nightfall users
    *
    * @async
    * @method makeTransfer
    * @param {UserMakeTransfer} options Object containing necessary data to perform transfers
```

Comment:
I know this was prob me, but let's have all `options` param equal

```suggestion
   * @param {UserMakeTransfer} options
```